isisd/isis_vty_fabricd.c

```diff
@@ -169,28 +170,31 @@ DEFUN (show_lsp_flooding,
 	if (argc == 4)
 		lspid = argv[3]->arg;
 
-	struct listnode *node;
+	struct listnode *node, *nnode, *inode;
 	struct isis_area *area;
+	struct isis *isis = NULL;
 
-	for (ALL_LIST_ELEMENTS_RO(isis->area_list, node, area)) {
-		struct lspdb_head *head = &area->lspdb[ISIS_LEVEL2 - 1];
-		struct isis_lsp *lsp;
+	for (ALL_LIST_ELEMENTS(im->isis, nnode, inode, isis)) {
```

Comment:
This is not a vrf all command. must be fixed

Reply:
Okay, will fix it.